chalice/package.py

```diff
@@ -101,8 +101,8 @@ def service_principal(self, service):
 class ResourceBuilder(object):
     def __init__(self,
                  application_builder,  # type: ApplicationGraphBuilder
-                 deps_builder,         # type: DependencyBuilder
-                 build_stage,          # type: BuildStage
+                 deps_builder,  # type: DependencyBuilder
```

Comment:
From what I can tell, `prcheck` still passes without these formatting changes.  I'm going to try reverting these changes and see if the CI passes.